test/runtests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+# runtests.sh
+
+# create bin directory if it doesn't exist
+if [ ! -d "../bin" ]
+then 
+	mkdir ../bin
```

Comment:
Maybe use `spaces` instead of `tab` here to be consistent with the `if` block below.